src/Codegen/Types.hack

```diff
@@ -119,10 +119,16 @@ function get_output_class(string $hack_type): ?string {
     try {
         $rc = new \ReflectionClass($hack_type);
 
-        $graphql_object = $rc->getAttributeClass(\Slack\GraphQL\ObjectType::class) ??
-            $rc->getAttributeClass(\Slack\GraphQL\InterfaceType::class);
+        $graphql_object = $rc->getAttributeClass(\Slack\GraphQL\ObjectType::class);
         if ($graphql_object) {
             return $graphql_object->getType();
+        } elseif (is_connection_type($rc)) {
```

Comment:
Connection types aren't annotated with GraphQL attributes so we need to check the separately.